### PR TITLE
[INLONG-2161][Manager] Add sort_id_params table for getClusterConfig interface

### LINKED ISSUE #2161: [Feature][InLong-Manager] Manager support getClusterConfig
### Description

[Feature][InLong-Manager] Manager support getClusterConfig 

### Use case

request URL         /api/inlong/manager/openapi/sort/standalone/getClusterConfig
requset method   GET
request params  ?clusterName=xxx&md5=xxx

example

image

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by imvan**
### Title Name: [INLONG-2161][Feature][Manager] Add sort_id_params table for getClusterConfig interface

Fixes #2161 

### Motivation

Add common id params table for sort cluster config interface

### Modifications

Add entity, mapper, service of common id params.
Add UT.

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+@Data
+@Builder
+public class SortIdParamsEntity implements Serializable {
+    private Integer id;
+    private String taskName;
+    private String type;
+    private String groupId;
+    private String streamId;
+    private static final long serialVersionUID = 1L;
+}
```
Comment (wardlican) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java:34]: Here, the default value of serialVersionUID is "1L", which is inappropriate and should be random

File: inlong-manager/manager-dao/src/main/resources/mappers/SortIdParamsEntityMapper.xml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  <insert id="insert" parameterType="org.apache.inlong.manager.dao.entity.SortIdParamsEntity">
+    insert into sort_id_params (id, task_name, type, 
+      group_id, stream_id)
+    values (#{id,jdbcType=INTEGER}, #{taskName,jdbcType=VARCHAR}, #{type,jdbcType=VARCHAR}, 
+      #{groupId,jdbcType=VARCHAR}, #{streamId,jdbcType=VARCHAR})
+  </insert>
+  <insert id="insertSelective" parameterType="org.apache.inlong.manager.dao.entity.SortIdParamsEntity">
+    insert into sort_id_params
+    <trim prefix="(" suffix=")" suffixOverrides=",">
+      <if test="id != null">
```
Comment (wardlican) [inline inlong-manager/manager-dao/src/main/resources/mappers/SortIdParamsEntityMapper.xml:55]: Shouldn't Id be generated automatically here? Why should assign an ID value when inserting?

Comment (imvan) [reply inlong-manager/manager-dao/src/main/resources/mappers/SortIdParamsEntityMapper.xml:55]: These codes are auto-generated by mybatis generator plugins. When insert one record, the feild "id" should not be assign.

**Review by wardlican — APPROVED** [review]

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java (hunk 1 above)
Comment (imvan) [reply inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java:34]: Auto generated by mybatis generator. 
I think Inlong must remain compatible with different release version, the value of serialVersionUID should be as easy as possible.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sort/SortClusterConfigResponse.java [hunk 3]
```diff
@@ -33,14 +33,18 @@
...
         String clusterName;
         List<SortTaskConfig> sortTasks;
     }
 
+    @Builder
+    @Data
     public static class SortTaskConfig {
         String taskName;
-        String type;
+        String sinkType;
```
Comment (luchunliang) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sort/SortClusterConfigResponse.java:47]: sinkType can be enumeration

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import lombok.Data;
+
+import java.io.Serializable;
+
+@Data
+@Builder
+public class SortIdParamsEntity implements Serializable {
+    private Integer id;
+    private String taskName;
+    private String type;
```
Comment (luchunliang) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java:30]: What is the type of id params?

Comment (luchunliang) [reply inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java:30]: String/integer or the others.

File: inlong-manager/manager-web/sql/apache_inlong_manager.sql [hunk 5]
```diff
@@ -1146,4 +1146,21 @@ CREATE TABLE `sort_cluster_config`
 ) ENGINE = InnoDB
   DEFAULT CHARSET = utf8mb4 COMMENT ='Sort cluster config table';
 
+-- ----------------------------
+-- Table structure for id_params
+-- ----------------------------
+DROP TABLE IF EXISTS `sort_id_params`;
+CREATE TABLE `sort_id_params`
```
Comment (luchunliang) [inline inlong-manager/manager-web/sql/apache_inlong_manager.sql:1153]: The table name is not correct.
The table is the relation between sort task and inlong datastream, not about ID parameters of sort task.

**Review by luchunliang — APPROVED** [review]

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sort/SortClusterConfigResponse.java (hunk 3 above)
Comment (imvan) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sort/SortClusterConfigResponse.java:47]: fixed

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java (hunk 4 above)
Comment (imvan) [reply inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java:30]: id params is a List<Map<String, String >> type

File: inlong-manager/manager-web/sql/apache_inlong_manager.sql (hunk 5 above)
Comment (imvan) [reply inlong-manager/manager-web/sql/apache_inlong_manager.sql:1153]: it's more like a common field of all id params, some extension params will be defined in tables like sort_id_params_kafka_ext,  sort_id_params_pulsar_ext

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java (hunk 4 above)
Comment (imvan) [reply inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java:30]: Please see SortClusterConfigResponse.java

Comment (imvan) [reply inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java:30]: it's common fields of id params, other fields will be defined in extension tables

**Review by healchow — APPROVED** [review]

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+package org.apache.inlong.manager.dao.entity;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.io.Serializable;
+
+@Data
+@Builder
+public class SortIdParamsEntity implements Serializable {
```
Comment (healchow) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/SortIdParamsEntity.java:27]: Wouldn't it be better to change the entity name to SortParam?
This table not only stores the ID of the Sort but can also be extended to store other parameters of the Sort.

BTW, the table name already has a plural meaning, so there is no need to use the plural form. 
For example, all user information is stored in the `user` table, and the table name is called `user`, not `users`.

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**